### PR TITLE
Change the group ID and rename the sub modules

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/1623

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -4,8 +4,8 @@ name = "jballerina.java.arrays"
 version = "@toml.version@"
 
 [[platform.java11.dependency]]
-path = "../jballerina.java.arrays-test-utils/build/libs/jballerina.java.arrays-test-utils-@project.version@.jar"
-groupId = "org.ballerinalang.stdlib.jballerina.java.arrays.testutils"
+path = "../test-utils/build/libs/jballerina.java.arrays-test-utils-@project.version@.jar"
+groupId = "io.ballerina.stdlib.jballerina.java.arrays.testutils"
 artifactId = "jballerina.java.arrays-test-utils"
```
Comment (ThisaruGuruge) [inline ballerina/Ballerina.toml:9]: We can remove these entries altogether

File: build.gradle [hunk 2]
```diff
@@ -24,7 +24,7 @@ plugins {
 
 
 allprojects {
-    group = 'org.ballerinalang'
+    group = 'io.ballerina'
```
Comment (ThisaruGuruge) [inline build.gradle:27]:
```suggestion
    group = project.group
```

File: gradle.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 org.gradle.caching=true
-group=org.ballerinalang
+group=io.ballerina
```
Comment (ThisaruGuruge) [inline gradle.properties:2]:
```suggestion
group=io.ballerina.stdlib
```

**Review by ThisaruGuruge — COMMENTED** [review]
Let's update the `ballerinaGradlePluginVersion` to `0.10.0` too. It will correct the test coverage reduction.

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (ThisaruGuruge) [inline ballerina/Ballerina.toml:9]: We can remove these entries altogether

Comment (ThisaruGuruge) [reply ballerina/Ballerina.toml:9]: Sorry, my bad. Ignore this comment

File: gradle.properties (hunk 3 above)
Comment (BuddhiWathsala) [reply gradle.properties]: Fixed in https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/195/commits/853a715b9e2b754ebddd75a2af46cde9780a6ead

File: build.gradle (hunk 2 above)
Comment (BuddhiWathsala) [reply build.gradle]: Fixed in https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/195/commits/853a715b9e2b754ebddd75a2af46cde9780a6ead

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**